packages/bitcore-wallet-service/src/config.ts

```diff
@@ -63,6 +62,9 @@ module.exports = {
     defaultProvider: 'BitPay',
     fetchInterval: 60, // in minutes
   },
+  maintenanceOpts: {
+    bwsIsLive: true,
```

Comment:
maybe is better to set  'maintenanceMode: true' / false
so if unset (as it will from upgrading in current installations), 
the server will continue to work

Reply:
Thanks, I've now made those changes.

Reply:
In the latest commit I had to make some important changes, I've added error handling to Bitcore Wallet Client while Bitcore Wallet Service is under maintenance.  

Secondly I moved the root handler for all requests while bws is under maintenance under CORS policy, as the order in which the handlers were called was causing problems sending error responses to bwc.